# This fixes an incorrect autodetection of NeoverseV1 architecture at build time
# Fixes https://github.com/EESSI/software-layer/pull/310#issuecomment-1734345487
--- cpuid_arm64.c.orig  2022-02-20 21:35:05.000000000 +0000
+++ cpuid_arm64.c       2023-09-26 16:27:15.265602255 +0000
@@ -73,8 +73,8 @@
   "TSV110",
   "EMAG8180",
   "NEOVERSEN1",
-  "NEOVERSEV1"
-  "NEOVERSEN2"
+  "NEOVERSEV1",
+  "NEOVERSEN2",
   "THUNDERX3T110",
   "VORTEX",
   "CORTEXA55",
